### PR TITLE
ci(): reproduction templates + sandbox deploy

### PR DESCRIPTION
**Pull request by ShaMan123**
## Description

- updated reproduction templates in the issue template for v6 and v5
- added the beta tag to the npmpublish workflow so the version dropdown of the issue template reflects it
- updated the sandbox deploy script to use the branch of fabric that deployed it

## Changes

- extracted git command logic from scripts/index to scripts/git
-

### CODE REVIEW

File: .codesandbox/deploy.mjs [hunk 1]
```diff
@@ -39,8 +40,11 @@ export async function createCodeSandbox(appPath) {
   );
   // omit linked package
   if (packageJSON.dependencies.fabric.startsWith('file:')) {
-    packageJSON.dependencies.fabric = '*';
+    const { repo, branch } = getGitInfo();
+    packageJSON.dependencies.fabric = `${repo}#${branch}`;
```
Comment (ShaMan123) [inline .codesandbox/deploy.mjs:44]: installs from github 
great for PRs
I guess CodesandboxCI handles this under the hood but for now this is great

**Review by ShaMan123 — COMMENTED** [review]
ready to merge

**Review by ShaMan123 — COMMENTED** [review]
done!

**Review by ShaMan123 — COMMENTED** [review]
future proofed for next majors!

**Review by asturur — APPROVED** [review]

**Comment by ShaMan123** [comment]
That was easy!

**Comment by ShaMan123** [comment]
I overlooked that fact that we need to keep the old repro templates for v5
pushing now

**Review by asturur — APPROVED** [review]

**Pull request merged**